nxdrive/utils.py

```diff
@@ -1286,3 +1289,47 @@ def compute_digest(path: Path, digest_func: str, callback: Callable = None) -> s
         return UNACCESSIBLE_HASH
 
     return str(h.hexdigest())
+
+
+def config_paths() -> Tuple[List[Path], Path]:
```

Comment:
Well, you can change the signature to return a `Tuple` of `Path`s instead of casting the return to a `list`.